neon_tts_plugin_tacotron2/__init__.py

```diff
@@ -76,13 +69,10 @@ def get_tts(self, sentence: str, output_file: str, speaker: Optional[dict] = Non
         # if request_lang.lower() == "zh-zh":
         #     request_lang = "cmn-cn"
 
-        to_speak = format_speak_tags(sentence)
```

Comment:
Will this model handle `<speak>` and `</speak>` tags (possibly without matched pairs)?